packages/harper-ls/package.yaml

```diff
@@ -0,0 +1,45 @@
+---
+name: harper-ls
+description: The Grammar Checker for Developers
+homepage: https://github.com/chilipepperhott/harper-ls
+licenses:
+  - Apache-2.0
+languages:
+  - Markdown,
+  - Rust,
+  - TypeScript,
+  - JavasCript,
+  - Python,
+  - Go,
+  - C,
+  - C++,
+  - Ruby,
+  - Swift,
+  - C#,
+  - TOML,
+  - Lua,
+categories:
+  - LSP
+
+source:
+  # renovate:versioning=loose
+  id: pkg:github/chilipepperhott/harper-ls@v0.7.3
```

Comment:
due to the change of the repo location, this is invalid and will result on the download failing